docker-app/qfieldcloud/filestorage/view_helpers.py

```diff
@@ -233,6 +235,32 @@ def download_field_file(
     http_host = request.headers.get("host", "")
     https_port = http_host.split(":")[-1] if ":" in http_host else "443"
 
+    download_range = request.headers.get("Range", "")
+    if download_range:
+        range_match = re.match(r"bytes=(\d+)-(\d+)?", download_range)
```

Comment:
What if you move this to some of the utils, something like "def parse_range(str) -> tuple[int, int | None]"?